pages/establishment/dashboard/views/index.jsx

```diff
@@ -30,33 +30,49 @@ const Index = ({
     pelh,
     nprc
   },
-  allowedActions
-}) => (
-  <Fragment>
-    <Header title={name} />
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-two-thirds">
-        <PanelList
-          panels={links.filter(link => allowedActions.includes(link.permissions)).map(link => <DashboardLink key={link.path} { ...link } />)}
-        />
+  allowedActions,
+  profile: {
+    asruAdmin
+  }
+}) => {
+
+  return (
+    <Fragment>
+      <Header title={name} />
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <PanelList
+            panels={links.filter(link => allowedActions.includes(link.permissions)).map(link => <DashboardLink key={link.path} { ...link } />)}
+          />
+        </div>
+        <Sidebar>
+          <dl>
+            <dt><Snippet>establishmentLicenceNumber</Snippet></dt>
+            <dd>{ licenceNumber }</dd>
+
+            {
+              pelh && <ProfileLink type="pelh" profile={pelh} />
+            }
+            {
+              nprc && <ProfileLink type="nprc" profile={nprc} />
+            }
+          </dl>
+        </Sidebar>
+        <Sidebar>
+          <dl>
+            <dt><Snippet>inspectors</Snippet></dt>
+            { asruAdmin && <dd><Link page="establishment.asru.inspectors" label={ <Snippet>pages.edit</Snippet> } /></dd> }
+
+            <dt><Snippet>spoc</Snippet></dt>
+            { asruAdmin && <dd><Link page="establishment.asru.spocs" label={ <Snippet>pages.edit</Snippet> } /></dd> }
+          </dl>
+        </Sidebar>
       </div>
-      <Sidebar>
-        <dl>
-          <dt><Snippet>establishmentLicenceNumber</Snippet></dt>
-          <dd>{ licenceNumber }</dd>
-
-          {
-            pelh && <ProfileLink type="pelh" profile={pelh} />
-          }
-          {
-            nprc && <ProfileLink type="nprc" profile={nprc} />
-          }
-        </dl>
-      </Sidebar>
-    </div>
-  </Fragment>
-);
+    </Fragment>
+  )
+  ;
+};
 
-const mapStateToProps = ({ static: { establishment, allowedActions } }) => ({ establishment, allowedActions });
+const mapStateToProps = ({ static: { establishment, allowedActions, profile } }) => ({ establishment, allowedActions, profile });
```

Comment:
If the only property you need from the profile is `asruAdmin` then it makes sense to only pass that.
```suggestion
const mapStateToProps = ({ static: { establishment, allowedActions, profile } }) => ({ establishment, allowedActions, asruAdmin: profile.asruAdmin });
```

Reply:
done